query-coordinator/src/main/scala/com/socrata/querycoordinator/QueryParser.scala

```diff
@@ -12,17 +12,17 @@ import com.socrata.soql.exceptions.SoQLException
 import com.socrata.soql.functions.SoQLFunctions
 import com.socrata.soql.parsing.{AbstractParser, Parser}
 import com.socrata.soql.typed._
-import com.socrata.soql.types.SoQLType
-import com.socrata.soql.{BinaryTree, Compound, Leaf, PipeQuery, SoQLAnalysis, SoQLAnalyzer, ast}
+import com.socrata.soql.types.{SoQLType, SoQLValue}
+import com.socrata.soql.{AnalysisContext, BinaryTree, Compound, Leaf, ParameterSpec, PipeQuery, SoQLAnalysis, SoQLAnalyzer, ast}
 import com.typesafe.scalalogging.Logger
 import org.joda.time.DateTime
 
 
-class QueryParser(analyzer: SoQLAnalyzer[SoQLType], schemaFetcher: SchemaFetcher, maxRows: Option[Int], defaultRowsLimit: Int) {
+class QueryParser(analyzer: SoQLAnalyzer[SoQLType, SoQLValue], schemaFetcher: SchemaFetcher, maxRows: Option[Int], defaultRowsLimit: Int) {
   import QueryParser._ // scalastyle:ignore import.grouping
   import com.socrata.querycoordinator.util.Join._
 
-  type AnalysisContext = Map[String, DatasetContext[SoQLType]]
+  type AnalysisContext = com.socrata.soql.AnalysisContext[SoQLType, SoQLValue]
```

Comment:
weird, it looks like this was fully imported above

Reply:
It was, but it needs to be qualified here because we're hiding the imported name.